dubbo-common/src/main/java/org/apache/dubbo/config/RegistryConfig.java

```diff
@@ -220,12 +219,6 @@ public RegistryConfig(ApplicationModel applicationModel, String address, String
         setProtocol(protocol);
     }
 
-    @Override
-    @Parameter(key = REGISTRY_CLUSTER_KEY)
-    public String getId() {
-        return super.getId();
-    }
-
```

Comment:
Please keep this method and only remove `@Parameter(key = REGISTRY_CLUSTER_KEY)`.

Reply:
Fixed.